### PR TITLE
✨ add catapulta txs

### PR DESCRIPTION
**Pull request by MartinGbz**
## Recap
- Count all txs fees from Catapulta address: `0x020E4359255f907DF480EbFfc8a7b7beac0c0216`
- Remove `swapToVariable` count because DEPLOYER_21 ran all the `swapToVariable` method, and because we already taking into account all tx from DEPLOYER_21, no need to count additionally these txs

### CODE REVIEW

File: .env.example
Comment (NandyBa) [inline .env.example:1]: Can be changed but remember to use good start block for eth refund (for proposal generation)

File: index.js [hunk 1]
```diff
@@ -35,12 +43,14 @@ async function main() {
   }
   await getOtherInteractions();
   console.log("Other interactions fetched ✅");
-  await getSwapTovariableStats();
-  console.log("Swap to variable stats fetched ✅");
+  // await getSwapTovariableStats(); // deployer 21 is used for every swap to variable so we already have all the txs from it
```
Comment (NandyBa) [inline index.js]: So maybe delete this or you prefer to keep it and deleted in other PR for easy search ?

**Review by NandyBa — APPROVED** [review]

File: .env.example
Comment (MartinGbz) [reply .env.example:1]: When we will have to generate it we will double check the start & end block 👍

File: index.js (hunk 1 above)
Comment (MartinGbz) [reply index.js]: Yeah, I keep it if one day we have to use it again. But the `getSwapTovariableStats` remains here in any case so I can remove it if you want

**Pull request merged**